### PR TITLE
Convert gasPriceMultiplier to BigNumber

### PR DESCRIPTION
**Pull request by vponline**
Small fix to change `gasPriceMultiplier` into a **BigNumber** before using in calculations.

### CODE REVIEW

File: packages/airnode-utilities/src/evm/gas-prices/gas-prices.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ export const parsePriorityFee = ({ value, unit }: PriorityFee) =>
   ethers.utils.parseUnits(value.toString(), unit ?? 'wei');
 
 export const multiplyGasPrice = (gasPrice: BigNumber, gasPriceMultiplier: number) =>
-  gasPrice.mul(Math.round(gasPriceMultiplier * 100)).div(BigNumber.from(100));
+  gasPrice.mul(BigNumber.from(Math.round(gasPriceMultiplier * 100))).div(BigNumber.from(100));
```
Comment (Siegrift) [inline packages/airnode-utilities/src/evm/gas-prices/gas-prices.ts:18]: Why is this needed? Afaik BigNumber accepts regular numbers as well.

Comment (vponline) [reply packages/airnode-utilities/src/evm/gas-prices/gas-prices.ts:18]: This is mostly for consistency (pointed out in #1043) as all the other numbers are converted to BigNumber in gas-prices (and mostly elsewhere in Airnode I believe).

**Review by acenolaza — APPROVED** [review]
Thanks 😁
